lib/mumuki/domain/status/submission/passed.rb

```diff
@@ -8,4 +8,8 @@ def self.passed?
   def self.iconize
     {class: :success, type: 'check-circle'}
   end
+
+  def self.exp_given
```

Comment:
Why are we using `exp` instead of `experience`?

Reply:
`exp` and `xp` are common shorthands for `experience points` - you are not awarding experience, but experience points.

See https://en.wikipedia.org/wiki/Experience_point

I could change it to `experience_points_given` if needed...

Reply:
ok!